build.gradle

```diff
@@ -31,7 +31,7 @@ dependencies {
     androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
-    compile 'com.android.support:appcompat-v7:23.1.1'
+    compile "com.android.support:appcompat-v7:${project.supportLibVersion}"
```

Comment:
I was moving the version numbers in another PR differently.
But this is ok, we just have to check make sure the other PR works too.
I will add the project extensions in that one to avoid issues for now.